src/pages/JournalentryPage.js

```diff
@@ -0,0 +1,17 @@
+import JournalEntry from '../components/JournalEntry';
+import Heading from '../components/Heading';
+import {loadFromLocalStorage} from '../util/localstorage';
+import {useState} from 'react';
+
+export default function CalendarPage() {
+  const [data, setData] = useState(loadFromLocalStorage('JournalEntry') ?? []);
```

Comment:
If you don't need `setData`, you don't have to destructure it:
```suggestion
  const [data] = useState(loadFromLocalStorage('JournalEntry') ?? []);
```
But if so, you don't need a state here, but you could use a normal `const` as well:
```suggestion
  const data = loadFromLocalStorage('JournalEntry') ?? []);
```